test/AsmJs/rlexe.xml

```diff
@@ -2,6 +2,18 @@
 <regress-exe>
   <test>
     <default>
+      <files>argTest.js</files>
+      <compile-flags>-maic:0</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>argTestMixed.js</files>
```

Comment:
The new tests are fine, I'm just thinking maybe existing tests already covering this? The reason we didn't hit this was just because we were not running all existing tests (`-simdjs` issue). If you change `runtests.py`, remove that flag, maybe you can see the difference?